instrumentation/github.com/gin-gonic/gin/otelgin/config.go

```diff
@@ -51,20 +51,30 @@ var defaultSpanNameFormatter SpanNameFormatter = func(c *gin.Context) string {
 
 // Filter is a predicate used to determine whether a given http.request should
 // be traced. A Filter must return true if the request should be traced.
+// Deprecated: use GinFilter instead.
+// v0.61.0 is the last version to support this.
+// v0.62.0 will remove this type.
```

Comment:
I am not sure if we need/should document these kind of things.

Reply:
removed